apps/src/sites/studio/pages/levels-standalone_video2-main.js

```diff
@@ -0,0 +1,27 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore} from '@cdo/apps/redux';
+import LabContainer from '@cdo/apps/code-studio/components/LabContainer';
+import MusicLabView from '@cdo/apps/music/views/MusicView';
+import ProjectContainer from '@cdo/apps/labs/projects/ProjectContainer';
+import {logError} from '@cdo/apps/music/utils/MusicMetrics';
+import {getStandaloneProjectId} from '@cdo/apps/labs/projects/utils';
+
+$(document).ready(function () {
```

Comment:
Interesting, so seems like we will still need to create one of these entry points for any new lab, but can then switch to any other lab from there?

Reply:
Yes, that is the principle.  It's still early on this, but I think we could use a new, common piece of code when either of these level types is encountered in a script level.

Reply:
That common piece of code is now in place.